cis-k8s-job/README.md

```diff
@@ -1,13 +1,37 @@
 # AccuKnox CIS K8s Job
+Instruction to perform CIS Benchmark on Cluster via AccuKnox CIS K8s Job
 
-## Helm install
+### Prerequisites:
+- [Docker](https://docs.docker.com/engine/install/)
+- [Helm (v3.13.1 or later)](https://v3.helm.sh/docs/intro/install/)
 
+### Parameters:
+| Variable             | Sample Value           | Description                |
+| -------------------- | ---------------------- | -------------------------- |
+| accuknox.url         | cspm.demo.accuknox.com | AccuKnox CSPM Endpoint URL |
```

Comment:
```suggestion
| accuknox.url         | cspm.demo.accuknox.com | AccuKnox API Endpoint URL |
```

Reply:
Since we're using two AccuKnox API endpoints as follows
 - cwpp.demo.accuknox.com
 - cspm.demo.accuknox.com
I've used CSPM keyword